.circleci/config.yml

```diff
@@ -19,3 +19,31 @@ jobs:
       - run: go get github.com/Masterminds/glide
       - run: glide install --strip-vendor
       - run: make build_linux
+
+  docker_push_master:
+    working_directory: /go/src/github.com/aerogear/aerogear-app-metrics
+    docker:
+      - image: circleci/golang:1.9
+    steps:
+      - checkout
+      - run: make setup
+      - setup_remote_docker
+      - run: make build_linux
+      - run: make docker_build
+      - run: make docker_push_latest
```

Comment:
If I pull any image from a registry without specifying a tag, `latest` will be pulled and in most cases I would always expect `latest` to be a stable release.

The result of this PR would be that every time a PR is merged to master there will be a new version of `aerogear/ups-config-operator:latest`. There's nothing wrong with this but are you sure this is what you want? There's a potential that `latest` will be unstable.

This is not a change request, just making sure you understand the implications before merging :)

Reply:
lastest to me means: master - a real tag is more "stable" to me

Reply:
Sure, but the practice of using latest to represent the latest stable release **is common convention**. Some examples:

https://hub.docker.com/r/openshift/origin/tags/
https://hub.docker.com/r/library/postgres/
https://hub.docker.com/_/node/
https://hub.docker.com/_/golang/
... many more

Not saying it's wrong but it is nice to follow conventions. If you're interested in an alternative approach, checkout the aerogear-app-metrics repo We push a `master` tag every build on master. and then with releases we push `latest` and the version number (`x.y.z`).

Either way it's up to you!

Reply:
@matzew @darahayes I'm ok with pushing a `-master` tag (instead of latest) by default and do the 'real' release tags manually.